2022-01-15 15:45:06,943 INFO    Thread-12 :23868 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project', 'run_id': '3z86yztt', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/sweep-y32fxp96/config-3z86yztt.yaml', 'sweep_id': 'y32fxp96'}
2022-01-15 15:45:06,944 INFO    Thread-12 :23868 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:45:06,944 INFO    Thread-12 :23868 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154506-3z86yztt/logs/debug.log
2022-01-15 15:45:06,944 INFO    Thread-12 :23868 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154506-3z86yztt/logs/debug-internal.log
2022-01-15 15:45:06,945 INFO    Thread-12 :23868 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:45:06,945 INFO    Thread-12 :23868 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 150, 'drop_out': 0.15, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
config: {}
2022-01-15 15:45:06,945 INFO    Thread-12 :23868 [wandb_init.py:init():460] starting backend
2022-01-15 15:45:06,945 INFO    Thread-12 :23868 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:45:06,951 INFO    Thread-12 :23868 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:45:06,956 INFO    Thread-12 :23868 [backend.py:ensure_launched():221] started backend process with pid: 23958
2022-01-15 15:45:06,959 INFO    Thread-12 :23868 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:45:06,962 INFO    Thread-12 :23868 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 150, 'drop_out': 0.15, 'epochs': 5, 'lr': 0.001, 'optimizer': 'adam'}
2022-01-15 15:45:06,967 INFO    Thread-12 :23868 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:45:07,028 INFO    Thread-12 :23868 [wandb_init.py:init():563] communicating current version
2022-01-15 15:45:10,468 INFO    Thread-12 :23868 [wandb_init.py:init():568] got version response 
2022-01-15 15:45:10,469 INFO    Thread-12 :23868 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:45:10,804 INFO    Thread-12 :23868 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:45:15,813 INFO    Thread-12 :23868 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:45:15,814 INFO    Thread-12 :23868 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:45:15,815 INFO    Thread-12 :23868 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:45:15,816 ERROR   Thread-12 :23868 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:45:15,816 INFO    Thread-12 :23868 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:45:19,269 INFO    Thread-12 :23868 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/3z86yztt
2022-01-15 15:45:19,269 INFO    Thread-12 :23868 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:45:19,271 INFO    Thread-12 :23868 [wandb_run.py:_restore():1752] restore
2022-01-15 15:45:19,276 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-01-15 15:45:19,979 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-01-15 15:45:20,857 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 18066
}

2022-01-15 15:45:20,964 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 18066
}

2022-01-15 15:45:21,068 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 18066
}

2022-01-15 15:45:21,182 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 18066
}

2022-01-15 15:45:21,289 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 18066
}

2022-01-15 15:45:21,397 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:21,507 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:21,616 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:21,734 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:21,839 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:22,427 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}

2022-01-15 15:45:23,212 INFO    Thread-12 :23868 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18066
  total_bytes: 18066
}
local_info {
}

2022-01-15 15:45:24,385 INFO    Thread-12 :23868 [wandb_run.py:_append_files():2180] logging synced files
